keyboards/gggw/crosses/3x5/rev1/keyboard.json

```diff
@@ -0,0 +1,84 @@
+{
+    "manufacturer": "Good Great Grand Wonderful LLC",
+    "keyboard_name": "Crosses 3x5",
+    "maintainer": "vinniefranco",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP26", "GP22", "GP21", "GP23", "GP20"],
+        "rows": ["GP27", "GP28", "GP29", "GP9"]
+    },
+    "processor": "RP2040",
+    "split": {
+        "enabled": true,
+        "serial": {
+            "driver": "vendor",
+            "pin": "GP4"
+        },
+        "transport": {
+            "sync": {
+                "layer_state": true,
+                "matrix_state": true
+            }
+        }
+    },
+    "tapping": {
+        "term": 195
+    },
```

Comment:
This should be left to the end user to configure. It also doesn't differ from the default in a noticeable manner.
```suggestion
```